libevmasm/Assembly.h

```diff
@@ -142,6 +142,9 @@ class Assembly
 		std::map<std::string, unsigned> const& _sourceIndices = std::map<std::string, unsigned>()
 	) const;
 
+	static std::vector<size_t> decodeSubIds(u256 data);
+	static u256 encodeSubIds(std::vector<size_t> const& subIds);
```

Comment:
Why are these helpers needed/better than the original code?

Reply:
These changes are helping access deeper in sub assembly. For example from original object we can access sub sub object. Without this I would need to copy sub sub object